### PR TITLE
Use composer bin-dir

### LINKED ISSUE #3: Use composer bin-dir
In `InstallHooksScript` the script must use the bin-dir of composer instead of the hardcoded `bin`

### PR DESCRIPTION
**Pull request by nclsHart**
I needed to remove the composer hooks because in this particular case, the bin dir is not the same and I don't see a proper way to fix this. 

Fix #3.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,10 +21,6 @@
         "symfony/var-dumper": "~2.8|~3.0",
         "composer/composer": "^1.2"
     },
-    "scripts": {
```
Comment (notFloran) [inline composer.json]: I think that the following setting can fix our issue:

```
    "config": {
        "sort-packages": true,
        "bin-dir": "bin/"
    },
```

Composer will use our `bin/` directory and so he will find the hook.

**Comment by notFloran** [comment]
@nclsHart have you tried your code in one of our projects (like "si") ?

File: composer.json (hunk 1 above)
Comment (nclsHart) [reply composer.json]: It was part of my "not so clean possible solution" :D. But ok, I'll fix that.

**Comment by nclsHart** [comment]
@notFloran Nop. Only on php-git-hooks itself before removing the hook. I'll do some monkey test on another project.
We need to add some tests on this project.

**Comment by notFloran** [comment]
I test the branch with 2 projets without issues

**Pull request merged**